model/commitqueue/github_pr_sender.go

```diff
@@ -164,6 +166,37 @@ func (s *githubPRLogger) sendPatchResult(msg *GithubMergePR, pr event.PRInfo) {
 	s.statusSender.Send(c)
 }
 
+func SendFailedMessageToPR(owner, repo string, prNumber int) error {
+	env := evergreen.GetEnvironment()
+	sender, err := env.GetSender(evergreen.SenderGithubStatus)
+	if err != nil {
+		return errors.Wrap(err, "error getting environment")
+	}
+	ghToken, err := env.Settings().GetGithubOauthToken()
+	if err != nil {
+		return errors.Wrap(err, "error getting github oauth token")
+	}
+
+	ctxWithCancel, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	pr, err := thirdparty.GetGithubPullRequest(ctxWithCancel, ghToken, owner, repo, prNumber)
```

Comment:
an alternative to getting the PR from GitHub could be to save the head SHA in the patch when we create it
https://github.com/evergreen-ci/evergreen/blob/72262b72b08104da687e13ce48fd7d76ff029be9/model/patch/patch.go#L401

Reply:
It sounds like the owner is wrong too though. It looks like we get the PR from github in a lot of places so I'm not convinced sending around more data in patches is better than getting the PR again.

Reply:
why is the owner not the same?
https://github.com/evergreen-ci/evergreen/blob/72262b72b08104da687e13ce48fd7d76ff029be9/model/patch/patch.go#L404

Reply:
I would argue the more information we fill in for a patch's `GithubPatchData` the less confusing the code will be (so you don't have to keep track of what's been set), but I don't have a strong opinion.
I would argue against getting the PR if we don't need to to avoid GitHub API limits.

Reply:
Okay I'm convinced haha